acquisitions/src/main/resources/thunderjet/mod-consortia/features/consortium.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Consortium object in mod-consortia api tests
+
+  Background:
+    * print karate.info.scenarioName
+
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+  Scenario: Create, Read, Update a consortium
+
+    # Post consortium
+    Given path '/consortia'
+    And request
+    """
+    {
+      id: '111841e3-e6fb-4191-8fd8-5674a5107c32',
+      name: 'Test'
+    }
+    """
+    When method POST
+    Then status 201
```

Comment:
Please add addintional test that we could not create second consortium

Reply:
Ok